components/pkg-export-helm/src/values.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate habitat_pkg_export_docker as export_docker;
+
+use std::io::Write;
+
+use export_docker::Result;
+
+pub struct Values {
+    values: Vec<ValuesEntry>,
+}
+
+impl Values {
+    pub fn new() -> Self {
+        Values { values: Vec::new() }
+    }
+
+    pub fn add_entry(&mut self, variable: &str, value: &str) {
+        self.values.push(ValuesEntry {
+            variable: variable.to_owned(),
+            value: value.to_owned(),
+        });
+    }
+
+    pub fn generate(&self, write: &mut Write) -> Result<()> {
+        let mut out = "".to_owned();
+        for entry in &self.values {
+            out = out + &format!("{}: \"{}\"\n", entry.variable, entry.value);
```

Comment:
Is this snippet supposed to be a part JSON document? Does it need JSON-specific escaping?

Reply:
don't think so. It's more toml. You can see the syntax here: https://docs.helm.sh/developing_charts/#values-files

Reply:
"A values file is formatted in YAML"

Reply:
Alright, so it's YAML. But still, I'm wondering if `entry.variable` and `entry.value` should be escaped. Just to avoid a scenario where `variable` is `foo`, and value is `bar"\nbaz: "quux`, so the result without escaping would be (for a single entry):
```
foo: "bar"
baz: "quux"
```

Reply:
yeah but it's escapped already through the "" around the value, isn't it?

Reply:
ah ok, i get your meaning now. will look into it

Reply:
looking at various docs, I failed to find how to handle double-quotes (") in the values. In any case, these values are coming from the Manifest (k8s exporter) so that should ensure they are all as expected (some of them are already being checked i think).